flake8.ini

```diff
@@ -0,0 +1,40 @@
+[flake8]
+
+# Plugins
+use-flake8-tabs = True
+## The following are replaced by flake8-tabs plugin, reported as ET codes rather than E codes.
+# E121, E122, E123, E126, E127, E128,
+## The following (all disabled) are not replaced by flake8-tabs,
+# E124 - Requires mixing spaces and tabs: Closing bracket does not match visual indentation.
+# E125 - Does not take tabs into consideration: Continuation line with same indent as next logical line.
+# E129 - Requires mixing spaces and tabs: Visually indented line with same indent as next logical line
+# E131 - Requires mixing spaces and tabs: Continuation line unaligned for hanging indent
+# E133 - Our preference handled by ET126: Closing bracket is missing indentation
+
+
+# Reporting
+statistics = True
+doctests = True
+show-source = True
+
+# Options
+max-complexity = 15
+max-line-length = 110
+# Final bracket should match indentation of the start of the line of the opening bracket
+hang-closing = False
+
+ignore =
+	W191,  # indentation contains tabs
+	W503,  # line break before binary operator. We want W504(line break after binary operator)
```

Comment:
According to https://flake8.pycqa.org/en/latest/user/configuration.html comments should not be on the same line